app/js/components/common/table.jsx

```diff
@@ -77,6 +77,7 @@ class cohortTable  extends Component {
         {this.state.toDisplay.length > 0 ?
           <div className="table">
             <h1 className="text-center">{this.state.description}</h1>
+            <h5 className="text-center">This search has {this.state.searchHistory.length} result(s)</h5>
```

Comment:
You can also deconstruct `this.state` to reduce the line length.

Reply:
Thank you for that observation. Done!